news/2017-11-05-Official-Mapping-Contests:-osu!-Mapping-Olympiad.md

```diff
@@ -0,0 +1,71 @@
+
+Let me elaborate on the concept for the osu! Mapping Olympiad. To ensure that there is a healthy and consistent flow of official contests, 
+we have worked on a concept which involved all types of modes - three contests per mode will happen annually in a specific order. 
+To keep things interesting, we have also worked on different kind of contests - so stay tuned in the upcoming months!
+
+### Songs (STILL NEED HELP WITH THE SONG PREVIEWS)
+
+To start off, we are launching this contest with not one, but **three** songs for you to tackle! Showcasing a variety of our Featured Artists, have yourself a listen to the tracks you’ll be taking on:
+
+- Kuba Oms - Beautiful Uncertainty (Redux)
+- OISHII - Honey Snowflakes
+- cySmix - Escapism
+
+There’s going to be something for everyone’s taste - funky rhythms by Kuba Oms, fast-paced goodness by OISHII and some chill beats by cYsmix!
+
+### Ruleset
+
+- This contest is for the **osu!** (standard) mode only.
+- Each entry should follow the Ranking Criteria.
+- Beatmap submissions may contain files up to 15 MB.
+- Storyboarding will not be considered in the judging.
+- You may submit **one** difficulty for **one** song. Multiple submissions for multiple songs will not be accepted.
+- You may collaborate if you like, but only the submitter of each entry will receive credit for judging and prizes.
+- Maps submitted to the contest must not be uploaded to the BSS until the contest has concluded and the winners have been announced.
+- **Send your finished beatmap as .osu-file to XXX before the deadline expired. Any submissions received after the deadline expired will not be accepted.**
+
+### Judging Criteria
+
+We have also slightly reworked the judging criteria to provide a more reasonable judging base. Down below you can see what your map will be judged for:
+
+- Musical Relevance (10P) - How well does the entry follow the music with its patterns, its structure and overall concept?
+- Creativity (10P) - How unique and/or innovative is the entry in its use of concepts and overall design?
+- Technique (10P) - How well does the entry utilize the editor and its tools in its creation? 
+- Impression (10P) - How does the judge view the map in its entirety?
+- Hitsounding (5P) - How well does the entry utilize hitsounding to emphasize and follow the music?
+
+**Speaking of Judging: We are still looking for capable judges! Are you interested? Feel free to drop a PM to Irreversible with your motivation, knowledge and possible references, where you've already judged. As a compensation, you will receive 2 months of supporter. Please be aware that we can only accept people who do have enough ressources (knowledge, time, reliability) to successfully do this.**
+
+### Prizes
+
+After the entries have been judged and the results finalized, each of the top three maps for each track will receive the following prizes:
```

Comment:
each of the maps implies that all the maps get all 3 places of prizes, should probably be reworded to say that the three best entries qualify for the following prices or something among those lines

Reply:
Each map will have its own 1st 2nd 3rd and each receive the prizes listed

Reply:
ah no my bad i read that wrong and got confused, might probably be worth being clearer though because i needed to read chaos' comment and the sentence in question multiple times to get this correctly into my head.